GymBro/Views/RoutineViews/RExerciseView_AddExerciseSheet.swift

```diff
@@ -11,14 +11,24 @@
 
 import SwiftUI
 
-struct AddExerciseButton: View {
+/// Exercise Picker in RExerciseView 
+///
+/// Using the wheel picker to select the exercise. 
+/// User can also add new exercise by clicking New button.
+struct RExerciseView_AddExerciseSheet: View {
```

Comment:
@cheniwei Remember to add the View documentation in the future. I leave the easy documentation for `RExerciseView` are it's derived sub views. I didn't notice when #7 .😅😅

Reply:
I would recommend to seperate the code changes of different issue in different PRs in the future. Fixing multiple issues in one PR will make it harder to debug.

Reply:
Agree! 
I will make sure to focus on one issue at once!

Reply:
It's better to do that. You can just keep the code changes relative to #issue15.

Reply:
Will do!

Reply:
@alex308248 done fixing all the PR issues! Tks for the valuable experience sharing!! It helps a lot! Please review the PR again when you are available. 🍕